plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/SwtBotAppEngineActions.java

```diff
@@ -65,9 +66,15 @@ public static IProject createNativeWebAppProject(SWTWorkbenchBot bot, String pro
       bot.textWithLabel("Java package:").setText(javaPackage);
     }
     // can take a loooong time to resolve jars (e.g. servlet-api.jar) from Maven Central
-    int libraryResolutionTimeout = 60000/* ms */;
+    int libraryResolutionTimeout = 300 * 1000/* ms */;
     SwtBotTimeoutManager.setTimeout(libraryResolutionTimeout);
-    SwtBotTestingUtilities.clickButtonAndWaitForWindowClose(bot, bot.button("Finish"));
+    try {
+      SwtBotTestingUtilities.clickButtonAndWaitForWindowClose(bot, bot.button("Finish"));
+    } catch (TimeoutException ex) {
+      System.out.println("FATAL: timed out while waiting for the wizard to close. Cannot recover. "
+          + "This may havoc all subsequent integration tests: "
+          + "https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/1925.");
+    }
```

Comment:
If the test could continue, it would normally fail. However, in reality the workbench will pop up a message dialog (https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/1925#issuecomment-302800050), so this test hangs before it could ever tear down normally. After 10 minutes, JUnit (or whoever) will kill the integration test bundle itself. I think we should really wait enough let the wizard close itself to avoid the pop-up.